assets/js/type-defs/contexts.js

```diff
@@ -215,4 +215,34 @@
  * @property {number}                  currentPostId      The post ID being edited.
  */
 
+/**
+ * @typedef {Object} ValidationContext
+ *
+ * @property {function(string):string}  getValidationError       Return
```

Comment:
The column widths really make this harder to read :)

Reply:
Agreed. I've added another editor ruler at the `100` column in my IDE so that when 80 is hit for comments and it's like this I can extend the comments to the 100 ruler instead (which I've applied as a commit here).